src/source/reporting.rst

```diff
@@ -0,0 +1,285 @@
+
+If we add a secondary dimension, for example *Category*, the resulting view could be:
+
+==============  ==============  =========  ===========
+  DIMENSION     DIMENSION        METRIC     METRIC
+--------------  --------------  ---------  -----------
+*Country*       *Category*      *Clicks*    *ViewCTR*
+==============  ==============  =========  ===========
+Netherlands     Bicycles         356        0.5678
+Netherlands     Camping Gear     174        0.455
+Netherlands     Mobile Phones    546        0.0245
+ . . .             . . .         . . .       . . .
+Spain           Bicycles         987        0.123
+Spain           Camping Gear     5537       0.2311
+Spain           Mobile Phones    987        0.123
+ . . .             . . .         . . .       . . .
+France          Bicycles         12743      0.2357
+France          Camping Gear     753        0.1124
+France          Bicycles         882        0.3575
+==============  ==============  =========  ===========
+
+The dimensions supported by the reporting API have discrete values (like *Netherlands* or *Bicycles*), and the data is split by each such value, over which statistics (like *Clicks* or *Impressions*) are calculated.
+
+Supported Dimensions
+********************************************
+
+=================  =========  =============================
+Name                 Type      Description
+=================  =========  =============================
+``am:date``         String     The date when the events happened. See `Time Aggregation`_
+``am:adID``         Long       ID of the ad(s)
+``am:CPC``          Long       The Cost Per Click (in Cents) of the ad(s)
+``am:categoryID``   Long       The category ID of the ad(s)
+``am:regionID``     Long       The region ID of the ad(s)
+=================  =========  =============================
+
+.. note:: *Date* is a special dimension, in that you can specify the granularity of the timeseries breakdown. In other words, data is aggregated over units of time (such as days, weeks, months or years) when calculating metrics over it. See `Time Aggregation`_ for options on granularity.
+
+
+Supported Metrics
+********************************************
+
+================================  =========  =========  ============================================================
+Name                                Type     `Scope`_    Description
+================================  =========  =========  ============================================================
+``am:clicks``                      Long       Hit        Number of clicks
+``am:impressions``                 Long       Hit        Nnumber of impressions
+``am:websiteClicks``               Long       Hit        Number of website clicks (leads)
+``am:emails``                      Long       Hit        Number of emails
+``am:engagements``                 Long       Hit        Number of engagements. Calculation: ``am:websiteClicks + am:emails``
```

Comment:
Oh I can remove it, it will simplify the tables... I noticed google is explicit about it https://developers.google.com/analytics/devguides/reporting/core/dimsmets#view=detail&group=adwords&jump=ga_ctr

Reply:
Or do you mean leave it out only for engagements, because it's an open bucket of event types